src/mono/wasm/debugger/BrowserDebugProxy/ValueTypeClass.cs

```diff
@@ -110,17 +111,39 @@ JObject GetFieldWithMetadata(FieldTypeClass field, JObject fieldValue, bool isSt
                 return fieldValue;
             }
         }
+        public async Task<string> ToString(MonoSDBHelper sdbAgent, CancellationToken token)
```

Comment:
nit: rename to `InvokeToStringAsync`, or just `ToStringAsync`.

Reply:
Done